.github/workflows/postgres.yml

```diff
@@ -57,7 +56,8 @@ jobs:
         cp config/database.yml.sample config/database.yml
         cp config/initializers/contact_us.rb.example config/initializers/contact_us.rb
         cp config/initializers/wicked_pdf.rb.example config/initializers/wicked_pdf.rb
-        cp config/credentials.yml.enc.workflow config/credentials.yml.enc
+        # generate a default credential file and key
+        bundle exec EDITOR='echo "$(cat config/credentials.yml.example)" >' rails credentials:edit
```

Comment:
Nice. That should always guarantee that the key and credentials work for the test system